Messy.py

```diff
@@ -0,0 +1,252 @@
+"""Submit jobs to HPC
+
+Returns:
+    _type_: _description_
+"""
+
+import sys
+import os
+import time
+import getpass
+import subprocess
+from shutil import which
+from stat import S_ISDIR, S_ISREG
+import paramiko
+import jinja2
+
+# setup logging
+paramiko.util.log_to_file("Messy.log")
+
+
+class Messy:
+    """Submit jobs to SLURM cluster"""
+
+    def __init__(self, username: str, hostname: str, port: int) -> None:
+        self.username = username
+        self.hostname = hostname
+        self.port = port
+        self.password = self.get_password()
+        self.transport = self.set_transport()
+        self.ssh_client = self.set_ssh_client()
+        self.sftp_client = self.set_sftp_client()
+
+    def get_password(self) -> str:
+        """Sets user password"""
+        password = getpass.getpass(
+            f"password for {self.username}@{self.hostname}: "
+        )
+        return password
+
+    def set_transport(self) -> paramiko.Transport:
+        """
+        Creates an paramiko transport
+        """
+        transport = paramiko.Transport((self.hostname, self.port))
+        return transport
+
+    def set_ssh_client(self) -> paramiko.SSHClient:
+        """
+        Creates an SSH client and connects to remote server
+        """
+        ssh_client = paramiko.SSHClient()
+        ssh_client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        ssh_client.connect(
```

Comment:
Maybe include an option to allow key-based authentication.

Reply:
it is not a priority at the moment. I created an issue about this.